File: hubsmsa-backend/hubsmsa-persistence/src/main/java/br/gov/pbh/prodabel/hubsmsa/enumeration/StatusEnum.java
package br.gov.pbh.prodabel.hubsmsa.enumeration;


/***
 * Enum com o estado do status - A - Ativo ou I - Inativo 
 */
public enum StatusEnum {
	
	A ("Sim"),
	I ("Não");


	private String name;

	private StatusEnum(String name) {
		this.name = name;
	}

	public String getName() {
		return name;
	}
	
}

File: hubsmsa-frontend/src/app/shared/shared-models/enum/status.enum.ts
export enum StatusEnum {
    ATIVO = 'A',
    INATIVO = 'I'
}

export const statusEnumMensagem = {
    [StatusEnum.ATIVO]: 'Ativo',
    [StatusEnum.INATIVO]: 'Inativo',
};

export const statusEnumMensagemVisualizar = {
    [StatusEnum.ATIVO]: 'Sim',
    [StatusEnum.INATIVO]: 'Não',
};